public/video-ui/src/components/VideoEdit/formComponents/VideoPoster.js

```diff
@@ -19,7 +19,7 @@ class VideoPosterImageEdit extends React.Component {
 
   render () {
 
-    if (!this.props.editMode) {
+    if (this.props.video && !this.props.editMode) {
```

Comment:
I think it will always be null when the page is loading so its not really invalid data?